### ISSUE TITLE
Torchscript file is not working with detect.py

### ISSUE THREAD

**Issue by RizwanMunawar** [ultralytics/yolov5#6000]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Detection

### Bug

Below, command is throwing error, when I am detecting with torchscript file.
`python .\detect.py --weights "B:\yolov5-master\yolov5s.torchscript" --source "B:\Testing videos\test.avi"`

![Screenshot 2021-12-16 111722](https://user-images.githubusercontent.com/62513924/146318648-e9715b60-63dc-434c-adbe-fc16f07886dc.png)
,

### Environment

Windows 10
Python 3.7
yolov5 latest release

### Minimal Reproducible Example

`python .\detect.py --weights "B:\yolov5-master\yolov5s.torchscript" --source "B:\Testing videos\test.avi"`

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@RizwanMunawar thanks for the bug report! I am able to reproduce this problem. I'll add a TODO to investigate today.

```
!python export.py --weights yolov5s.pt --include torchscript
!python detect.py --weights yolov5s.torchscript
```

**Comment by glenn-jocher** [comment]
@RizwanMunawar good news 😃! Your original issue may now be fixed ✅ in PR #6007. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by RizwanMunawar** [comment]
Cool, it works fine now.
